marketplace/deployer_util/config_helper.py

```diff
@@ -400,6 +408,38 @@ def type(self):
     return self._type
 
 
+class SchemaGcp:
+  """Accesses top level GCP constraints."""
+
+  def __init__(self, dictionary):
+    self._nodes = _maybe_get_and_apply(dictionary, 'nodes',
+                                       lambda v: SchemaNodes(v))
+
+  @property
+  def nodes(self):
+    return self._nodes
+
+
+class SchemaNodes:
+  """Accesses GKE cluster node constraints."""
+
+  def __init__(self, dictionary):
+    if 'requiredOauthScopes' in dictionary:
+      required_oauth_scopes = dictionary['requiredOauthScopes']
+      if not isinstance(required_oauth_scopes, list):
+        raise InvalidSchema('nodes.requiredOauthScopes must be a list')
+      for scope in required_oauth_scopes:
+        if not scope.startswith(_OAUTH_SCOPE_PREFIX):
+          raise InvalidSchema(
+              'OAuth scope references must be fully-qualified (start with {})'
+              .format(_OAUTH_SCOPE_PREFIX))
+      self._required_oauth_scopes = dictionary['requiredOauthScopes']
+
+  @property
+  def required_oauth_scopes(self):
+    return self._required_oauth_scopes
+
+
```

Comment:
can you help fix this line: https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/1262d4dcdaeccedee6b818777bf97e92dbd237dd/marketplace/deployer_util/config_helper.py#L883

`_ISTIO_TYPES` should be replaced by `valid_list`!